openreview/conference/templates/programchairWebfield.js

```diff
@@ -847,7 +848,7 @@ var calcMetaReviewCount = function(blindedNotes) {
 var calcDecisions = function(blindedNotes) {
   var finalDecisions = [];
   blindedNotes.forEach(function(note) {
-    if (note.details.decision) {
+    if (note.details.decision && note.details.decision !== 'No Decision') {
```

Comment:
I'm not sure when `note.details.decision` is a string

Reply:
when there's no decision i set it to the string 'No Decision'

Reply:
Could you instead create a decision object with 'No Decision'?

and there is no need to change openreview-web:

`var paperDecisions = _.find(submission.details.directReplies, ['invitation', getInvitationId(DECISION_NAME, submission.number)]) ?? [{ content: 'No Decision' }]`

Reply:
I would prefer to avoid that otherwise you need to check that everywhere.

Reply:
i think there was some difficulty with sorting when defaulting to object here but i can try again

Reply:
I don't think you need the second condition anymore, right?